js/main.js

```diff
@@ -0,0 +1,22 @@
+function getRandomInteger(min, max){
+  min = (min < 0) ? 0 : min;
+  max = (max > 0) ? max : 0;
```

Comment:
```suggestion
  min = min < 0 ? 0 : min;
  max = max > 0 ? max : 0;
```
Скобки тут ни к чему